app/components-test-lib/page.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { Box, Heading, VStack, useToast, Text, HStack } from "@chakra-ui/react";
+import MockButton from "../components/mock-button";
+
+const ComponentsTestLib = () => {
+  const toast = useToast();
+
+  if (process.env.NODE_ENV !== "development") {
```

Comment:
i actually think we can get rid of this because we kind of _do_ want this in "production" (or at least in the sandbox environment we setup for testing), that way non-eng stakeholders can look at the page and see what we've got going on